core/services/keystore/keys/starkkey/key.go

```diff
@@ -0,0 +1,104 @@
+package starkkey
+
+import (
+	crypto_rand "crypto/rand"
+	"encoding/hex"
+	"fmt"
+	"io"
+	"math/big"
+
+	"github.com/smartcontractkit/caigo"
+)
+
+// Raw represents the Stark private key
+type Raw []byte
+
+// Key gets the Key
+func (raw Raw) Key() Key {
+	k := Key{}
+	var err error
+
+	k.priv = new(big.Int).SetBytes(raw)
+	k.pub.X, k.pub.Y, err = caigo.Curve.PrivateToPoint(k.priv)
+	if err != nil {
+		panic(err) // key not generated
```

Comment:
is this panic ok? i know you are copy and pasting, but panic in the core node seems really dangerous

Reply:
@aalu1418 @archseer 
some of the other keys do the same, but I don't see why - it's not satisfying an interface, and the callers have the ability to return errors already :thinking: